pkg/coinbase/validator.go

```diff
@@ -33,12 +89,11 @@ func (v Validator) ToString() string {
 	)
 }
 
-func (c *Client) ListValidators(ctx context.Context, networkId string, assetId string) ([]Validator, error) {
-	validatorList, _, err := c.client.ValidatorsAPI.ListValidators(
-		ctx,
-		normalizeNetwork(networkId),
-		assetId,
-	).Execute()
+func (c *Client) ListValidators(ctx context.Context, networkId string, assetId string, status *ValidatorStatus) ([]Validator, error) {
```

Comment:
Status should be an optional filter if we can use the options paradigm here!

Reply:
Can do.

Reply:
Done.